src/components/contact.jsx

```diff
@@ -61,10 +63,21 @@ export const Contact = (props) => {
               target="_blank"
               rel="noopener noreferrer"
               href={props.data ? props.data.linkedin : "/"}
+              style={{ border: "none" }}
```

Comment:
Had this issue: https://github.com/FortAwesome/Font-Awesome/issues/6208

Basically, an underline was showing up on this <a> tag... but doing `text-decoration: none` did not fix it. Finally found this issue after like 20 mins of googling lol. Apparently `border: none` does the trick. Strange.

Reply:
Hover before:
![image](https://user-images.githubusercontent.com/13268990/150047771-c2363317-59f0-4f49-8055-977e228b5d12.png)

Hover now:
![image](https://user-images.githubusercontent.com/13268990/150047746-00a2bdb8-b186-441c-a189-c7ea3b4ba80a.png)